### PR TITLE
Refactoring and fixes

### PR DESCRIPTION
**Pull request by manuelsc**

### CODE REVIEW

File: src/app/components/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -222,219 +222,51 @@
...
+					[currency]="'GWEI'"
+					[unit]="unit"
+					[todayTooltip]="
+						'Today\'s consensus reward income. Note that this is counted in beaconchain days, a beaconchain day might start mid day depending on the network.'
+					"
+					[last7DaysTooltip]=""
+					[last31DaysTooltip]=""
+					[totalTooltip]="'Total (Contains your consensus rewards)'"
+					[aprTooltip]="
+						'APR (Estimated yearly returns on both consensus layer (staking) and execution layer (transaction fees))'
```
Comment (D13ce) [inline src/app/components/dashboard/dashboard.component.html:253]: **Bug:** The APR tooltip now is the same for all 3 categories.

File: src/app/components/dashboard/performance-item.component.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with Beaconchain Dashboard.  If not, see <http://www.gnu.org/licenses/>.
+ -->
+
+<div class="grid">
+	<div class="grid-cell left" (click)="unit.switchCurrencyPipe()">
+		<div class="grid">
+			<ion-label class="value left" [class]="performanceData.performance1d | valuestyle : 0">
+				{{ performanceData.performance1d | mcurrency : currency : unit.pref.Cons }}
```
Comment (D13ce) [inline src/app/components/dashboard/performance-item.component.html:24]: **Question:** This (and some other values) has been `unit.pref.Exec` before for the execution performance, see https://github.com/gobitfly/eth2-beaconchain-explorer-app/pull/262/files#diff-176bd5938e1566da4de6f22f275b3298a13f42789fea88d54bc04b5729085df5L374

Is this change intentional?

**Review by D13ce — CHANGES_REQUESTED** [review]

**Review by D13ce — APPROVED** [review]
lgtm

**Review by mostafalarki1368mos — APPROVED** [review]
oll

**Pull request merged**